src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -114,7 +135,7 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
             disabled,
             formInspector,
             onChange,
-            value,
+            value
```

Comment:
We have these trailing commas on purpose, because they'll result in smaller diffs when more values are added in this object.